velox/exec/HashProbe.cpp

```diff
@@ -1379,7 +1378,12 @@ void HashProbe::noMoreInputInternal() {
     }
     return;
```

Comment:
Is it possible to reach here with 'promises' being non-empty? If so, will we run into the warning again?